tools/system_test_jobs.groovy

```diff
@@ -0,0 +1,80 @@
+// Driver list
```

Comment:
COnsider adding a high-level description at the top of the file that gives context about what this does, etc.

Reply:
Added